client/views/monitorings/list_birds.html

```diff
@@ -79,6 +79,14 @@ <h1 class="page-header">
                      model="monitoringController.filter.species"></field>
             </div>
 
+            <div class="col-md-4 col-lg-3">
+              <field name="threat" type="single-choice" nomenclature="main_threats"
+                     label="{{'LIST_FILTER_THREAT' | translate}}"
+                     placeholder="- {{'INPUT_PLACEHOLDER_ALL' | translate}} -"
```

Comment:
One time binding only for performance
```suggestion
                     placeholder="- {{::'INPUT_PLACEHOLDER_ALL' | translate}} -"
```